### ISSUE TITLE
Changing phpstorm theme changes colors assigned in drawio document

### ISSUE THREAD

**Issue by samtlewis** [docToolchain/diagrams.net-intellij-plugin#44]
I created a document using the draw.io desktop app on Mac. Added it to my project in PHPStorm.
But when I open it within PHPStorm (set to Darcula theme), the colors assigned to the document elements get changed to match the theme.

For instance, I have a simple rectangle that was set to white fill. When opening it in PHPStorm using this plugin, the fill is now set to #2a2a2a which i'm guessing comes from the darcula theme. If I change the them from darcula to macos Light, the colors get changed back.

I don't think the contents of a diagram should be changed based on the IDE's theme changing. If multiple people viewed the document with different themes the content would be different.

**Comment by rdmueller** [comment]
interesting. Thanx for reporting this.
draw.io supports themes and we try to switch them in order to match the theme of the IDE. I though this only applies to the background and foreground colors, not the fills.
Have to give it a try.

**Comment by ms-lolo** [comment]
I see #65 was closed and released but I'm not sure how to actually change the theme for diagrams.

![image](https://user-images.githubusercontent.com/61251739/158681319-3de7fac7-a185-445a-8bc0-5786e044149f.png)

I updated the scheme setting of the Diagrams section but you can see the background is still dark in my editor. I tried restarting the editor but nothing changed.

**Comment by ms-lolo** [comment]
Ah! Sorry! I think I found the right setting!

![image](https://user-images.githubusercontent.com/61251739/158683867-62808c98-527a-462b-b2da-fd073a62f583.png)

You can ignore my requests :)

**Comment by ahus1** [comment]
I wonder if the plugin should default to the light theme. Then a user can still change it to the dark theme if they like. WDYT?

**Comment by ms-lolo** [comment]
Yeah I personally think it should default to the light theme because the theme of the diagram represents how we want the artifact to look, not just the environment where we are creating the artifact. This is pretty different from our code theme where my choice of colors has no effect on how other people read the code. I can't author diagrams in a dark theme because the resulting PNG ends up being embedded on a light themed website.
